examples/siro_sandbox/README.md

```diff
@@ -63,7 +63,7 @@ If your FPS is very low, consider this workaround. This habitat-sim commit repla
 # Command-line Options
 
 ## App State and use cases
-Use `--app-state rearrange` to run rearrange task in the sandbox app. Supported app states: `rearrange` (default), `fetch`, `socialnav`, `free_camera`. These correspond to the different use cases for the HITL tool (see corresponding implementations in `app_states/app_state_<state name>.py` files). Note, `free_camera` is special: it's not a "task" state, it's a state that lets the user control the camera instead of controlling an agent (useful for debugging and policies' behaviour visualisation). Also, `tutorial` is a special state that shows a tutorial sequence at the start of every episode to introduce the user to the scene and goals in a human-in-the-loop context. Tutorial should be followed by the task app state (`rearrange`, `fetch` or `socialnav`) and is currently supported only for the rearrange task.
```

Comment:
I would move this above `Command-line Options` even though you are technically documenting a command-line option. It's really important for folks to fundamentally understand what we mean by use cases. I would rewrite this section a bit to emphasize what we mean by use case (essentially a mini-application; there should be no dependency between use cases, e.g. `app_state_rearrange.py` and `app_state_throw_fetch_vr.py` are isolated from each other).

This is probably a good spot to repeat our advice for users: if you're looking to customize the HITL tool for your own use case (e.g. custom data-collection), a good starting point is to create your own AppState class by copy-pasting from an existing one.

It's unfortunate/confusing that app states and use cases aren't 1-to-1, but I don't want to change that. You already do a good job of explaining the tutorial exception.